From e3babfda49f00b278ed35a407556535f1ed0d08d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert=20L=C3=B6hning?= <robert.loehning@qt.io>
Date: Thu, 18 Sep 2025 12:24:29 +0200
Subject: [PATCH 193/553] Fix test compilation with QT_NO_WIDGETS

Amends 68a5ec23f683c7778f587842fe35a47b545cf9d1 where the
test function was added outside the right #ifdef-block.

Pick-to: 6.9 6.8 6.5
Task-number: QTBUG-138678
Change-Id: If96132d724b3373d728c8b440f3d5c2473d44610
Reviewed-by: Eskil Abrahamsen Blomfeldt <eskil.abrahamsen-blomfeldt@qt.io>
(cherry picked from commit 5aecefc6a1549fca72d29dc793e6f1835e9a0095)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 tests/auto/gui/text/qtexttable/tst_qtexttable.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/auto/gui/text/qtexttable/tst_qtexttable.cpp b/tests/auto/gui/text/qtexttable/tst_qtexttable.cpp
index f7e0f788f17..5825f7e89ca 100644
--- a/tests/auto/gui/text/qtexttable/tst_qtexttable.cpp
+++ b/tests/auto/gui/text/qtexttable/tst_qtexttable.cpp
@@ -83,9 +83,9 @@ private slots:
 
     void columnWidthWithImage_data();
     void columnWidthWithImage();
-#endif
 
     void QTBUG138678_insertColumnAtStartWithRowspan();
+#endif
 
 private:
     QTextTable *create2x2Table();
@@ -1413,7 +1413,6 @@ void tst_QTextTable::columnWidthWithImage()
     QVERIFY(rightRect.left() > leftRect.right());
 #endif
 }
-#endif
 
 void tst_QTextTable::QTBUG138678_insertColumnAtStartWithRowspan()
 {
@@ -1457,6 +1456,7 @@ void tst_QTextTable::QTBUG138678_insertColumnAtStartWithRowspan()
     // Don't assert
     currentTable->insertColumns(0, 1);
 }
+#endif
 
 
 QTEST_MAIN(tst_QTextTable)
-- 
2.51.2

